lib/page-loader.js

```diff
@@ -18,7 +18,7 @@ export default class PageLoader {
     if (route[0] !== '/') {
       throw new Error('Route name should start with a "/"')
     }
-    route = route.replace(/index$/, '')
+    route = route.replace(/\/index$/, '/')
```

Comment:
Before this received `/abcindex` and produced `/abc`
Now it only matches `/index` or `/abc/index`